streams/src/test/java/org/apache/kafka/streams/processor/internals/DefaultStateUpdaterTest.java

```diff
@@ -922,7 +914,6 @@ private void verifyRestoredActiveTasks(final StreamTask... tasks) throws Excepti
                 VERIFICATION_TIMEOUT,
                 "Did not get all restored active task within the given timeout!"
             );
-            assertTrue(restoredTasks.stream().allMatch(task -> task.state() == State.RESTORING));
```

Comment:
It doesn't make sense to verify the state of a mock task, because the mock will always return what we specified it should return independently of what the state updater does with the task. Totally missed that until now 🙂 !